docs/source/installation.rst

```diff
@@ -35,6 +35,26 @@ To install an extra simply add it in brackets like so::
     $ pip install rstcheck[sphinx,toml]
 
 
+Pre-commit Hook with Extras
+---------------------------
+
+To use ``rstcheck`` in a Git pre-commit hook with optional features:
+
+1. Install pre-commit::
+
+     pip install pre-commit
+
+2. Add to ``.pre-commit-config.yaml``::
+
+     - repo: https://github.com/rstcheck/rstcheck
+       rev: v6.2.4
```

Comment:
Please replace the version string with a placeholder like e.g. `<version>`.
This is to reduce the burden of the need to keep this part of the docs always up-to-date.